opencraft/settings.py

```diff
@@ -295,6 +295,10 @@
 # for provisioning Open edX app servers.
 DEFAULT_CONFIGURATION_VERSION = env('DEFAULT_CONFIGURATION_VERSION', default=DEFAULT_OPENEDX_RELEASE)
 
+# Git ref for stable Open edX release. Used as a default refspec for
+# configuration, edx-platform, forum, notifier, xqueue, and certs when creating production instances.
+OPENEDX_RELEASE_STABLE_REF = env('OPENEDX_RELEASE_STABLE_REF', default='named-release/dogwood.rc')
```

Comment:
@smarnach These questions remain open for you:

> Should we use the branch or the tag, at the end? Also, I don't see a dogwood branch on https://github.com/open-craft/edx-platform yet - what branch/ref will we be using on our fork?

https://github.com/open-craft/opencraft/pull/85#discussion_r64007865